api/openapi-spec/swagger.json

```diff
@@ -4697,6 +4704,27 @@
      }
     }
    },
+   "v1.Input": {
+    "required": [
+     "bus",
```

Comment:
Bis is not required on disks, are we sure it should be required here? We set disks to `sata` by default. I guess we can also set it to `usb` by default.